packages/office-ui-fabric-react/src/common/_themeVariables.scss

```diff
@@ -0,0 +1,55 @@
+$ms-font-mediumPlus-WebkitFontSmoothing: "[theme:ms-font-mediumPlus-WebkitFontSmoothing, default: antialiased]";
+$ms-font-mediumPlus-fontSize: "[theme:ms-font-mediumPlus-fontSize, default: 15px]";
+$ms-font-mediumPlus-fontWeight: "[theme:ms-font-mediumPlus-fontWeight, default: 400]";
+$ms-font-large-fontFamily: "[theme:ms-font-large-fontFamily, default: 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
+$ms-font-large-MozOsxFontSmoothing: "[theme:ms-font-large-MozOsxFontSmoothing, default: grayscale]";
+$ms-font-large-WebkitFontSmoothing: "[theme:ms-font-large-WebkitFontSmoothing, default: antialiased]";
+$ms-font-large-fontSize: "[theme:ms-font-large-fontSize, default: 17px]";
+$ms-font-large-fontWeight: "[theme:ms-font-large-fontWeight, default: 300]";
+$ms-font-xLarge-fontFamily: "[theme:ms-font-xLarge-fontFamily, default: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
+$ms-font-xLarge-MozOsxFontSmoothing: "[theme:ms-font-xLarge-MozOsxFontSmoothing, default: grayscale]";
+$ms-font-xLarge-WebkitFontSmoothing: "[theme:ms-font-xLarge-WebkitFontSmoothing, default: antialiased]";
+$ms-font-xLarge-fontSize: "[theme:ms-font-xLarge-fontSize, default: 21px]";
+$ms-font-xLarge-fontWeight: "[theme:ms-font-xLarge-fontWeight, default: 100]";
+$ms-font-xxLarge-fontFamily: "[theme:ms-font-xxLarge-fontFamily, default: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
+$ms-font-xxLarge-MozOsxFontSmoothing: "[theme:ms-font-xxLarge-MozOsxFontSmoothing, default: grayscale]";
+$ms-font-xxLarge-WebkitFontSmoothing: "[theme:ms-font-xxLarge-WebkitFontSmoothing, default: antialiased]";
+$ms-font-xxLarge-fontSize: "[theme:ms-font-xxLarge-fontSize, default: 28px]";
+$ms-font-xxLarge-fontWeight: "[theme:ms-font-xxLarge-fontWeight, default: 100]";
+$ms-font-superLarge-fontFamily: "[theme:ms-font-superLarge-fontFamily, default: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
+$ms-font-superLarge-MozOsxFontSmoothing: "[theme:ms-font-superLarge-MozOsxFontSmoothing, default: grayscale]";
+$ms-font-superLarge-WebkitFontSmoothing: "[theme:ms-font-superLarge-WebkitFontSmoothing, default: antialiased]";
+$ms-font-superLarge-fontSize: "[theme:ms-font-superLarge-fontSize, default: 42px]";
+$ms-font-superLarge-fontWeight: "[theme:ms-font-superLarge-fontWeight, default: 100]";
+$ms-font-mega-fontFamily: "[theme:ms-font-mega-fontFamily, default: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif]";
+$ms-font-mega-MozOsxFontSmoothing: "[theme:ms-font-mega-MozOsxFontSmoothing, default: grayscale]";
```

Comment:
This one is particularly weird. I wonder if @phkuo is reading comments remotely. :)

Reply:
Last I checked, The Most Magical Place On Earth(tm) did not include code reviews :'(

The theme variable names (that come after `theme:`) should be legal JavaScript variable names (since we parse a JSON object to get the theme values for these).
The SASS variable name looks fine though, it follows the conventions.